mantle/kola/tests/misc/selinux.go

```diff
@@ -118,25 +113,9 @@ func getSelinuxBooleanState(c cluster.TestCluster, m platform.Machine, seBool st
 	return boolState, nil
 }
 
-// SelinuxEnforce checks that some basic things work after `setenforce 1`
-func SelinuxEnforce(c cluster.TestCluster) {
-	cmdList := []cmdCheckOutput{
-		{"getenforce", true, "Enforcing"},
-		{"sudo setenforce 0", false, ""},
-		{"getenforce", true, "Permissive"},
-		{"sudo setenforce 1", false, ""},
-		{"getenforce", true, "Enforcing"},
-		{"systemctl --no-pager is-active system.slice", true, "active"},
-		{"sudo cp /etc/selinux/config{,.old}", false, ""},
-		{"sudo sed -i 's/SELINUX=permissive/SELINUX=enforcing/' /etc/selinux/config", false, ""},
-	}
-
-	m := c.Machines()[0]
+// SelinuxEnforce is verified by the 
+// https://github.com/coreos/fedora-coreos-config/blob/testing-devel/tests/kola/selinux/enforcing
```

Comment:
we don't even need this comment. Can drop it out.

Reply:
This suggestion was [implemented](https://github.com/coreos/coreos-assembler/commit/5e3c0f69d94649f20bb10f34426eb55fcdf19dae)